graviti/manager/common.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""Common tools."""
+
+import warnings
+from typing import TYPE_CHECKING
+
+from graviti.exception import StatusWarning
+
+if TYPE_CHECKING:
+    from graviti.manager.commit import Commit
+
+
+def check_dataset_status(head: "Commit", revision_name: str, revision_commit_id: str) -> None:
```

Comment:
Is `check_head_status` better?

Reply:
`revision_name` -> `remote_revision`
`revision_commit_id` -> `remote_commit_id`